src/items/functions/item/item_parse.cpp

```diff
@@ -11,6 +11,7 @@
 
 #include "items/functions/item/item_parse.hpp"
 #include "utils/pugicast.h"
+#include "item_parse.hpp"
```

Comment:
It is not necessary, its inclusion is already at the beginning of the file. (line 12)